about/glossary.rst

```diff
@@ -143,11 +162,14 @@ Glossary
    NIDEM
       National Intertidal Digital Elevation Model, a Digital Earth Australia product 
       derived from ITEM that maps the elevation relative to Mean Sea Level of the 
-      Australian intertidal zone. For more information see https://www.ga.gov.au/dea/products/nidem.
+      Australian intertidal zone. For more information see https://www.dea.ga.gov.au/products/dea-intertidal-elevation.
       
    NIR
       Near Infrared, referring to particular bands used to collect Earth observation 
       data
+  
+   NRT
+     Near-real time
```

Comment:
It may be worth extending this with something like '..., is a less refined/calibrated data 'Maturity Level', which is available significantly sooner after satellite acquisition and may be sufficient for many use cases.'

I'm not sure if here is a good spot for it, but a warning like: if your use case necessitates the highest data consistency, you'll likely want to instead only use ARD data. Along with this, please ensure you specify the required data maturity level in your query (to avoid consistency issues of having mixed maturity data in your in your results).

We may also want an entry for `Data Maturity Level` ?

Reply:
Added some extra info. There are some standard words on NRT somewhere, but I have no idea where they are. I will look around.